### PR TITLE
Add: label commands

### LINKED ISSUE #226: label update asdf (name fails with exception)  nil pointer dereference
When trying to update a label with a name it fails. 

```console
./harbor-cli label list       
┌────────────────────────────────────────────────────────────────────────────────────┐
│  ID      Name          Color         Description         Creation Time             │
│ ────────────────────────────────────────────────────────────────────────────────── │
│  4       asdf          #0095D3                           0 minute ago              │
└────────────────────────────────────────────────────────────────────────────────────┘

harbor-cli on  
❯ ./harbor-cli label update asdf
ERRO[0000] failed to parse label id: strconv.ParseInt: parsing "asdf": invalid syntax 
panic: runtime error: invalid memory address or nil pointer dereference
[signal SIGSEGV: segmentation violation code=0x2 addr=0x0 pc=0x1050802e0]

goroutine 1 [running]:
github.com/goharbor/harbor-cli/cmd/harbor/root/labels.UpdateLableCommand.func1(0x140001b3600?, {0x1400030b8c0?, 0x4?, 0x10508907a?})
        /Users/vadim/Development/container-registry/harbor-cli/cmd/harbor/root/labels/update.go:37 +0xf0
github.com/spf13/cobra.(*Command).execute(0x1400035cf08, {0x1400030b890, 0x1, 0x1})
        /Users/vadim/go/pkg/mod/github.com/spf13/cobra@v1.8.1/command.go:989 +0x81c
github.com/spf13/cobra.(*Command).ExecuteC(0x14000032c08)
        /Users/vadim/go/pkg/mod/github.com/spf13/cobra@v1.8.1/command.go:1117 +0x344
github.com/spf13/cobra.(*Command).Execute(0x1057d2758?)
        /Users/vadim/go/pkg/mod/github.com/spf13/cobra@v1.8.1/command.go:1041 +0x1c
main.main()
        /Users/vadim/Development/container-registry/harbor-cli/cmd/harbor/main.go:10 +0x20


```

### LINKED ISSUE #227: label update by name should be possible
It should be possible to change labels by their name, and name should be default. 

```console
./harbor-cli label update name

```

ID should be optional

### PR DESCRIPTION
**Pull request by Althaf66**
This fix #226 
This fix #227 
@amands98 @Vad1mo ,

**Created the new subcommands for Label**

`create` - for creating new labels
`delete` - for deleting existing labels
`list` - for listing all labels 
`update` - for updating existing labels

### CODE REVIEW

File: pkg/views/label/list/view.go [hunk 1]
```diff
@@ -0,0 +1,79 @@
...
+
+var columns = []table.Column{
+	{Title: "ID", Width: 6},
+	{Title: "Name", Width: 12},
+	{Title: "Color", Width: 12},
+	{Title: "Description", Width: 18},
+	{Title: "Creation Time", Width: 24},
+}
+
+func (m model) Init() tea.Cmd {
```
Comment (amands98) [inline pkg/views/label/list/view.go:27]: Just use base styling, there already the code in place to use that.

File: pkg/views/label/select/view.go [hunk 2]
```diff
@@ -0,0 +1,115 @@
...
+	tea "github.com/charmbracelet/bubbletea"
+	"github.com/goharbor/go-client/pkg/sdk/v2.0/models"
+	"github.com/goharbor/harbor-cli/pkg/views"
+)
+
+const listHeight = 14
+
+type item string
+
+func (i item) FilterValue() string { return "" }
```
Comment (amands98) [inline pkg/views/label/select/view.go:19]: here also

File: pkg/views/label/list/view.go (hunk 1 above)
Comment (Althaf66) [reply pkg/views/label/list/view.go:27]: @amands98, Thanks for the correction, I will change it accordingly

**Comment by Vad1mo** [comment]
looks good thank you

**Review by amands98 — APPROVED** [review]
LGTM :rocket:

**Review by bupd — APPROVED** [review]
@Althaf66 merge conflicts

Other than that LGTM 🎉.

**Comment by Althaf66** [comment]
ready for review

**Comment by bupd** [comment]
## Suggestions
```bash
❯ ./harbor label delete black
ERRO[0000] failed to delete label: [DELETE /labels/{label_id}][404] deleteLabelNotFound  &{Errors:[0xc0001b26e0]}
```

label deletion can be improved

**Pull request merged**